iolite/client.py

```diff
@@ -174,13 +172,16 @@ async def __response_handler(self, response: str, websocket) -> NoReturn:
 
         elif response_class == ClassMap.QuerySuccess.value:
             logger.info("Handling QuerySuccess")
+            if self.stop_event:
```

Comment:
I don't like this 😢 

I would rather have a way to resolve the response that was from a request and then we could control it better. It's a start but It's not the end goal